news/2023/2023-01-31-skinning-poll-2023.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "osu! 2023 Skinning Poll"
+date: 2023-01-31 17:00:00 +0000
+---
+
+Are you an avid skin creator who makes skins from scratch, or someone who makes remixed skins for fun? Maybe you are just a person who likes collecting skins and exploring what the community has to offer? Either way, we want your feedback on everything skinning related!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+With skinning continuously evolving, we created a survey through which the community can express their opinion on the current state of the skinning scene and help us better understand the community's expectations and needs.
+
+The link below will lead you to the survey page, while most questions are not mandatory to answer, the more feedback we receive, the easier it will be for us to make changes you want to see happen come to life.
+
+- [osu! 2023 Skinning Poll](https://docs.google.com/forms/d/e/1FAIpQLScEtYo-uxu-uSS7S3C_y7DV-EZWY47dfg9-YpUuThu8belCQw/viewform)
+
+The survey should only a couple of minutes to complete, so be sure to pass the news along to any of your friends that might be interested in skinning! Your opinion is extremely valuable to us.
+
+For those who wish to further engage with the skinning community, please feel free to take a look at the [skinning tutorial](https://skinship.xyz/tutorial/introduction) or check out the [skinning forum](https://osu.ppy.sh/community/forums/15)!
+
+There is currently no set date for announcing the survey results, but when the time comes it will be announced via the [skinship Discord server](https:/discord.skinship.xyz), and a write-up of the results will be released on the skinning forums.
```

Comment:
```suggestion
There is currently no set date for the announcement of the survey results, but when the time comes it will be announced via the [skinship Discord server](https:/discord.skinship.xyz), and a write-up will be released on the skinning forums.
```